### ISSUE TITLE
Great sharing ! One simple question though...

### ISSUE THREAD

**Issue by thusinh1969** [g2archie/UNET-MRI-RECONSTRUCTION#12]
I learn a lot from your sharing. The tf.image.ssim loss works well, and yes I chose BatchNorm for the regularization layer. Have not tried LayerNorm and Dropout.

My only question is when I look at your codes I do not see that you might have normalized the input/labels data (says /255.0 or something). Your saved data must have been normalized or you simply did not normalize deliberately? I always normalize and I have trained successfully.

Thank you.
Steve

**Comment by g2archie** [comment]
Hi Steve,

I am sorry that I haven't been monitoring my GitHub for a long time. The data were preprocessed for me and it has been normalized so I didn't have to do anything to it. Unfortunately, I cannot share the data. And I forgot to write a section to describe the data I used so it has caused some confusion. 

Hope it helps.

Thanks,

Wentao
